### ISSUE TITLE
Raml with included schemas break when those schemas include files too

### ISSUE THREAD

**Issue by Queueless** [StefH/RamlToOpenApiConverter#20]
Hello, 

I'm trying to convert the https://github.com/AMWA-TV/is-04 raml files to OAS and the tool is throwing an exception.  The problem is that the QueryAPI.raml includes a bunch of json schemas from a sub-folder "schemas", but those schema files also include from files in their own directory as well, but the YamlIncludeNodeDeserializer is expecting the includes to be in the root directory.

**Comment by StefH** [comment]
@Queueless 
Did you test the latest version?

**Comment by Queueless** [comment]
No, haven't.  I no longer work at the company that was utilizing this stuff
